src/api/tests/models/test_static.py

```diff
@@ -140,6 +140,77 @@ def test_statique_model_date_mise_en_service():
         StatiqueFactory.build(date_mise_en_service=tomorrow)
 
 
+def test_statique_model_restriction_gabarit_len():
+    """Test the Statique model `restriction_gabarit` field length."""
+    statique = StatiqueFactory.build(restriction_gabarit="Aucune")
+    assert statique.restriction_gabarit == "Aucune"
+
+    with pytest.raises(ValueError, match="String should have at least 2 characters"):
+        StatiqueFactory.build(restriction_gabarit="")
+
+
+def test_statique_model_str_fields_strip():
+    """Test the Statique model string fields stripping."""
+    nom_amenageur = " Foo Inc.  "
+    contact_amenageur = " contact@foo.org "
+    nom_operateur = " Bar Inc."
+    contact_operateur = "contact@bar.org  "
+    telephone_operateur = " tel:+33-1-44-27-63-50  "
+    nom_enseigne = " Big Company"
+    id_station_itinerance = "FR073P00001  "
+    id_station_local = " id-01  "
+    nom_station = " Station 01  "
+    adresse_station = " 1 baker street 75000 PARIS "
+    code_insee_commune = "74264 "
+    id_pdc_itinerance = "FR073E0042  "
+    id_pdc_local = " pdc-01 "
+    tarification = " It's Free!  "
+    horaires = " Open 24/7  "
+    restriction_gabarit = " None (for now)  "
```

Comment:
Also test with `"  "` in case length is calculated before stripping!

```suggestion
    restriction_gabarit = " None (for now)  "
```